### ISSUE TITLE
Hide top bars in GUI when menu is minimized

### ISSUE THREAD

**Issue by jacodg** [frankframework/frankframework#1194]
**Reporter**
Jaco de Groot

**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
When the menu is minimized (with the small arrow left at the bottom of the menu) to save space it would be nice to also hide the top bars with the FF! version and breadcrumb to save even more space.

**Describe alternatives you've considered**
None

**Additional context**
\-

**Comment by jacodg** [comment]
Discussed in core meeting: Collapse top bar to breadcrumb. Minimize space used for top bar.

**Comment by nielsm5** [comment]
When menu is folded in -> hide breadcrumbs, and top nav from 60 to 40 px. (and white background).

![Image](https://github.com/frankframework/frankframework/assets/4375703/c00b820b-8ec3-4050-895e-2e8172a653a8)

**Comment by philipsens** [comment]
From a UX perspective, a header should contain very imported functionalities and information. If it's in the way, hiding it isn't going to help.
Looking at the header, it is displaying: notifications, dtap, version, breadcrumb and time. When logged in, it also shows a logout button. 

When deciding to hide the top bars a few questions come to mind:
* Why do you want more space? Is the console to cramped?
* Is the info important enough to show at the top, if you are going to hide it?
* On a small screen, would it be logical to always hide the top bars?
* Is there another place when this information or these functionalities can be shown?

These (and possibly other) questions need to be answered before we can even think about how we are going to implement this. It is also good to consider alternatives, to provide a more broad view of the possibilities.

**Comment by jacodg** [comment]
I think the most important use case is when debugging with Ladybug. In that case it is nice to have as much space available as possible to show debug information. This has now been solved with the pop out button.
